src/snowflake/cli/_plugins/spcs/services/commands.py

```diff
@@ -297,6 +298,131 @@ def logs(
     return StreamResult(cast(Generator[CommandResult, None, None], stream))
 
 
+@app.command(requires_connection=True)
+def events(
+    name: FQN = ServiceNameArgument,
```

Comment:
Need to add description of the command in the docstring. This will be used by Typer to display help message. Use any working command as a reference. Like this:     https://github.com/snowflakedb/snowflake-cli/blob/1c2ef97810b3494119d3cae9d3d1106d227e15b8/src/snowflake/cli/_plugins/snowpark/commands.py#L132

Reply:
fixed.